lib/client.js

```diff
@@ -212,6 +212,7 @@ Client.prototype._setSequenceArgs = function(argsScheme, args) {
     return args;
   }
   for (var partIndex in argsScheme) {
+    partIndex = partIndex.replace('[]','')
```

Comment:
It happens when I try to use the string with the brackets. I don't know if the field can have brackets at all, but in my case it cannot. 
If i use the field without brackets, the field does not come to my args, due its difference to the field from the Scheme, with the brackets.